tests/phpunit/Mutator/Unwrap/UnwrapArrayChangeKeyCaseTest.php

```diff
@@ -41,14 +41,16 @@
 final class UnwrapArrayChangeKeyCaseTest extends AbstractMutatorTestCase
 {
     /**
-     * @dataProvider provideMutationCases
+     * @dataProvider mutationsProvider
+     *
+     * @param string|string[] $expected
      */
-    public function test_mutator($input, $expected = null): void
+    public function test_it_can_mutate(string $input, $expected = []): void
```

Comment:
What about changing `null` to `[]` inside `\Infection\Tests\Mutator\AbstractMutatorTestCase::doTest`?

Reply:
Done